### PR TITLE
perf: ⚡️ improt link constructor performance

### PR DESCRIPTION
**Pull request by JackieAnxis**

### CODE REVIEW

File: src/elements/element.ts [hunk 1]
```diff
@@ -23,7 +23,24 @@ export default class Element {
         const defaultConfigs = this.$_core.$_configs
 
         // override default style with user specified style in data
-        this.$_style = override(defaultConfigs[type].style, data.style)
+        // this.$_style = override(defaultConfigs[type].style, data.style)
+        this.$_style = defaultConfigs[type].style
+        if ('style' in data) {
+            for (let name of data.style as any) {
```
Comment (brickmaker) [inline src/elements/element.ts]: data.style is an object, not iterable

File: src/index.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { InteractionManager } from './interaction/interaction'
 import * as Utils from './utils/utils'
 import { Position } from './interfaces'
 import { EMPTY_FUNCTION } from './utils/const'
+import { link } from 'build/NetV'
```
Comment (brickmaker) [inline src/index.ts]: false import

**Review by brickmaker — CHANGES_REQUESTED** [review]
Some small issues, need change.
examples are broken too.

File: src/elements/element.ts (hunk 1 above)
Comment (JackieAnxis) [reply src/elements/element.ts]: fixed by commit 4084a8c

File: src/index.ts (hunk 2 above)
Comment (JackieAnxis) [reply src/index.ts]: fixed by commit 7f48d39

**Review by brickmaker — APPROVED** [review]

**Pull request merged**